### ISSUE TITLE
Error 258 when creating QR Code from Badge

### ISSUE THREAD

**Issue by El-Wollo** [beboxos/circuitpython#9]
Hi,
I'm getting an Error 258 when creating QR Code by pressing arrow down in Badge screen.
//
VERSION:3.0
END:VCARD

258
Error in badge selection
//
Error occurs after Line 1118.

Any sugestions on how to fix it?

**Comment by beboxos** [comment]
yes i have the same issue when url is complicated with some i unsupported char i think .

i'm currently searching a way with QR lib

**Comment by El-Wollo** [comment]
So when leaving the url out of teh vcard there should be no error?
In the url there was no special character. it was http://www.daimlertruck.com so nothing fancy here.

Keep Up the work. I like your project!
